techmatters/flex-plugins
# Conferencing: preserve dialog across states

## Description
This PR adds a `conferencing` reducer to our Redux store, that will keep track of the conferencing state for a given voice task. The idea is to preserve the partial input provided for the external call, in case the counselor change the view and comes back to the task.

Waiting on the core UI to wire this.

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1941)
- [n/a] Strings are localized
- [n/a] Tested for chat contacts

### Verification steps
- Start development server (`npm run dev`).
- Start a voice contact and accept in your end.
- Play around with the "add external party" dialog and confirm state is preserved if you change to a different view like Search (dialog kept open, partial input preserved).
- Confirm that a new participant placeholder is added when you add an external party. Note you can't interact with it until the party either accepts or rejects the call.
